modular_skyrat/modules/customization/modules/mob/living/carbon/carbon_update_icons.dm

```diff
@@ -103,11 +109,18 @@
 			var/datum/sprite_accessory/snouts/S = GLOB.sprite_accessories["snout"][dna.species.mutant_bodyparts["snout"][MUTANT_INDEX_NAME]]
 			if(S.use_muzzled_sprites && wear_mask.mutant_variants & STYLE_MUZZLE)
 				used_style = STYLE_MUZZLE
+		else if(dna.species.id == "teshari")
+			used_style = STYLE_TESHARI
+
 		switch(used_style)
 			if(STYLE_MUZZLE)
 				desired_icon = wear_mask.worn_icon_muzzled || 'modular_skyrat/master_files/icons/mob/clothing/mask_muzzled.dmi'
 			if(STYLE_VOX)
-				desired_icon = wear_mask.worn_icon_vox || 'modular_skyrat/master_files/icons/mob/clothing/mask_vox.dmi'
+				desired_icon = 'modular_skyrat/master_files/icons/mob/clothing/mask_vox.dmi'
```

Comment:
This is a regression, why are you forcing it to use the mask_vox.dmi file?

Reply:
i got no fuckin clue why i did that, mustve been a mistake